app/views/users/index.html.erb

```diff
@@ -20,7 +20,7 @@
   </div>
 </div>
 
-<div class="row to-member-cards">
+<div class="row">
```

Comment:
https://github.com/coopdevs/timeoverflow/blob/develop/spec/features/transfer_spec.rb#L55 you are probably affecting this test